powerVCF.psm1

```diff
@@ -1877,16 +1874,301 @@ Function Set-VCFCeip {
+
+Function Get-VCFBundles {
+<#
+    .SYNOPSIS
+    Get all Bundles i.e uploaded bundles and also bundles available via depot access
+	
+    .DESCRIPTION
+    Get all Bundles i.e uploaded bundles and also bundles available via depot access. 
+	
+    .EXAMPLE
+    This example gets the list of bundles and all details
+	
+	PS C:\> Get-VCFBundles
+
+    This example gets the list of bundles and filters on the version, download status and the id only
+	
+	PS C:\> Get-VCFBundles | Select version,downloadStatus,id 	
+#>
+
+    $headers = @{"Accept" = "application/json"}
+    $headers.Add("Authorization", "Basic $base64AuthInfo")
+    $uri = "https://$sddcManager/v1/bundles"
+
+    try { 	
+		    $response = Invoke-RestMethod -Method GET -URI $uri -headers $headers
+		    $response.elements      
+        }
+    catch {
+        # Call the function ResponseExeception which handles execption messages
+        ResponseExeception
+    }
+}
+Export-ModuleMember -Function Get-VCFBundles
+
+######### End Bundle Operations ##########
+
+######### Start Certificate Configuration Operations ##########
+
+Function Get-VCFCertificateAuthConfiguration {
+<#
+    .SYNOPSIS
+    Get certificate authorities information
+	
+    .DESCRIPTION
+     Retrieves the certificate authorities information for the connected SDDC Manager
+	
+    .EXAMPLE
+    This example shows how to get the certificate authority configuration from the SDDC Manager
+    you are connected to.
+	
+	PS C:\> Get-VCFCertificateAuthConfiguration
+#>
+
+    $headers = @{"Accept" = "application/json"}
+    $headers.Add("Authorization", "Basic $base64AuthInfo")
+    $uri = "https://$sddcManager/v1/certificate-authorities"
+
+    try { 	
+		    $response = Invoke-RestMethod -Method GET -URI $uri -headers $headers
+		    $response.elements
+        }
+    catch {
+        # Call the function ResponseExeception which handles execption messages
+        ResponseExeception
+    }
+}
+Export-ModuleMember -Function Get-VCFCertificateAuthConfiguration
+
+######### End Certificate Configuration Operations ##########
+
+######### Start Certificate Configuration Operations ##########
+
+Function Get-VCFDepotCredentials {
+<#
+    .SYNOPSIS
+    Get Depot Settings
+	
+    .DESCRIPTION
+     Retrieves the configuration for the depot of the connected SDDC Manager
+	
+    .EXAMPLE
+    This example shows credentials that have been configured for the depot.
+	
+	PS C:\> Get-VCFDepotCredentials	
+#>
+
+    $headers = @{"Accept" = "application/json"}
+    $headers.Add("Authorization", "Basic $base64AuthInfo")
+    $uri = "https://$sddcManager/v1/system/settings/depot"
+
+    try { 	
+		    $response = Invoke-RestMethod -Method GET -URI $uri -headers $headers
+		    $response
+        }
+    catch {
+        # Call the function ResponseExeception which handles execption messages
+        ResponseExeception
+    }
+}
+Export-ModuleMember -Function Get-VCFDepotCredentials
+
+Function Set-VCFDepotCredentials {
```

Comment:
Can this be used to update existing creds also? If so would be good to mention it in the synopsis

Reply:
Probably not as the method is PUT. Maybe another function for a PATCH then?

Reply:
It actually creates or overwrites based on testing